### ISSUE TITLE
Release of pertained weights

### ISSUE THREAD

**Issue by liuyuex97** [zztianzz/PF-Net-Point-Fractal-Network#6]
Thank you for sharing your work. Is it possible to release the pretained weights for ShapeNet sometime soon?

**Comment by zztianzz** [comment]
I have answer your questions by senting you an email using my email app. But it seems that you didn't received it. This is the content of the email： I am very sorry that  i can't go back to school now due to the coronavirus and i can't get the pretrain pth file.  Maybe it still need some times. Or you can ask researchers who have trained the model to share their pretrained pth files.

**Comment by liuyuex97** [comment]
Thank you for reaching out to me this way. I understand the situation -
it's the same for our lab as well. I am training on a remote server and can
also send you a copy of the trained .pth file if you want to post it on Git.

Best,
Louise

**Comment by antoalli** [comment]
Hi @liuyuex97, can you share the trained model?

**Comment by Onepiecesky** [comment]
@liuyuex97 ,Hi,Cloud you share the trained model? THX! email:609565170@qq.com.

**Comment by liuyuex97** [comment]
@antoalli @Onepiecesky  I have uploaded the trained generator model. It can be accessed through: https://cmu.box.com/s/owzs88ip84dti9a66sxbn5cbn3yfwfz5

Once the official authors released their pre-trained model, I will turn off the sharing. Please note this is not an official distribution. The model performance may not reflect the best performance presented in paper.

**Comment by lrrll** [comment]
> @antoalli @Onepiecesky I have uploaded the trained generator model. It can be accessed through: https://cmu.box.com/s/owzs88ip84dti9a66sxbn5cbn3yfwfz5
> 
> Once the official authors released their pre-trained model, I will turn off the sharing. Please note this is not an official distribution. The model performance may not reflect the best performance presented in paper.

hello it said the file might be corrupted and terminate called after throwing an instance of 'c10::Error'

**Comment by lrrll** [comment]
> @antoalli @Onepiecesky I have uploaded the trained generator model. It can be accessed through: https://cmu.box.com/s/owzs88ip84dti9a66sxbn5cbn3yfwfz5
> 
> Once the official authors released their pre-trained model, I will turn off the sharing. Please note this is not an official distribution. The model performance may not reflect the best performance presented in paper.

hello could you meet the problem about the dimension?

**Comment by liuyuex97** [comment]
@lrrll The weights I released were trained with an earlier version released by the authors in around Feb or March. I can try finding the source code and post it in my forked repo.

**Comment by lrrll** [comment]
thanks for your help; now I have complete it but in the last I could open the txt files in the test_example with mesh lab, but I see the author said it could, I don't know where is the problem, could you tell me? thank you very much!!!




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"liuyuex97"<notifications@github.com&gt;;
发送时间:&nbsp;2020年7月5日(星期天) 中午11:48
收件人:&nbsp;"zztianzz/PF-Net-Point-Fractal-Network"<PF-Net-Point-Fractal-Network@noreply.github.com&gt;;
抄送:&nbsp;"鹿茸蓉"<1009005627@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [zztianzz/PF-Net-Point-Fractal-Network] Release of pertained weights (#6)





 
@lrrll The weights I released were trained with an earlier version released by the authors in around Feb or March. I can try finding the source code and post it in my forked repo.

**Comment by liuyuex97** [comment]
The .txt files can be imported to Windows version of Meshlab. Just select import and specify the separating symbol between XYZ values when importing. Ubuntu version doesn't support .txt yet.
Alternatively I have been using Mayavi library for quick visualization. Its mlab.point3d can display pointcloud with specified XYZ values as numpy arrays. Here's a sample code I have:

```
from mayavi import mlab
fig = mlab.figure(bgcolor=(1, 1, 1), size=(640, 500))
mlab.point3d(x, y, z, color=(0.54, 0.82, 0.94), mode = "sphere", scale_factor = 0.03, figure=fig)
mlab.show()
```

**Comment by lrrll** [comment]
Due to my Windows version of Meshlab is not work, so i have achieved it by matlab. I have some questions about the code:(1)where achieve the choose of the input part of the point clouds;(2)how could i choose to&nbsp; visual the input of the point clouds and the result ,for example:the Table 1 and 2; the Fig 6 and 7 and 8 and 9. I am sorry to bother you ,i just learn about the point clouds, so i couldn't know the code absolutely, Thank you very much!




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"liuyuex97"<notifications@github.com&gt;;
发送时间:&nbsp;2020年7月9日(星期四) 上午8:02
收件人:&nbsp;"zztianzz/PF-Net-Point-Fractal-Network"<PF-Net-Point-Fractal-Network@noreply.github.com&gt;;
抄送:&nbsp;"鹿茸蓉"<1009005627@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [zztianzz/PF-Net-Point-Fractal-Network] Release of pertained weights (#6)





 
The .txt files can be imported to Windows version of Meshlab. Just select import and specify the separating symbol between XYZ values when importing. Ubuntu version doesn't support .txt yet.
 Alternatively I have been using Mayavi library for quick visualization. Its mlab.point3d can display pointcloud with specified XYZ values as numpy arrays. Here's a sample code I have:
 from mayavi import mlab fig = mlab.figure(bgcolor=(1, 1, 1), size=(640, 500)) mlab.point3d(x, y, z, color=(0.54, 0.82, 0.94), mode = "sphere", scale_factor = 0.03, figure=fig) mlab.show()

**Comment by liuyuex97** [comment]
@lrrll The newest release of show_recon.py in the official repo has a dataloader that feeds input data from the testset of ShapeNet to the generator model. You should be able to modify the completion setting like number of missing points to get results that resemble the figures.
The show_CD.py in the official repo also loads data from the testset. I haven't test the code personally but it would probably give you the CD loss shown in the tables.

**Comment by lrrll** [comment]
hello, recently i want to run the comparison-test, but i find the model of the three networks is not given,could you mind sharing them? thanks.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "鹿茸蓉"                                                                                    <1009005627@qq.com&gt;;
发送时间:&nbsp;2020年7月9日(星期四) 下午3:53
收件人:&nbsp;"zztianzz/PF-Net-Point-Fractal-Network"<reply@reply.github.com&gt;;

主题:&nbsp;回复： [zztianzz/PF-Net-Point-Fractal-Network] Release of pertained weights (#6)



Due to my Windows version of Meshlab is not work, so i have achieved it by matlab. I have some questions about the code:(1)where achieve the choose of the input part of the point clouds;(2)how could i choose to&nbsp; visual the input of the point clouds and the result ,for example:the Table 1 and 2; the Fig 6 and 7 and 8 and 9. I am sorry to bother you ,i just learn about the point clouds, so i couldn't know the code absolutely, Thank you very much!




------------------ 原始邮件 ------------------
发件人:&nbsp;"liuyuex97"<notifications@github.com&gt;;
发送时间:&nbsp;2020年7月9日(星期四) 上午8:02
收件人:&nbsp;"zztianzz/PF-Net-Point-Fractal-Network"<PF-Net-Point-Fractal-Network@noreply.github.com&gt;;
抄送:&nbsp;"鹿茸蓉"<1009005627@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [zztianzz/PF-Net-Point-Fractal-Network] Release of pertained weights (#6)





 
The .txt files can be imported to Windows version of Meshlab. Just select import and specify the separating symbol between XYZ values when importing. Ubuntu version doesn't support .txt yet.
 Alternatively I have been using Mayavi library for quick visualization. Its mlab.point3d can display pointcloud with specified XYZ values as numpy arrays. Here's a sample code I have:
 from mayavi import mlab fig = mlab.figure(bgcolor=(1, 1, 1), size=(640, 500)) mlab.point3d(x, y, z, color=(0.54, 0.82, 0.94), mode = "sphere", scale_factor = 0.03, figure=fig) mlab.show()

**Comment by KushanRaj** [comment]
> @antoalli @Onepiecesky I have uploaded the trained generator model. It can be accessed through: https://cmu.box.com/s/owzs88ip84dti9a66sxbn5cbn3yfwfz5
> 
> Once the official authors released their pre-trained model, I will turn off the sharing. Please note this is not an official distribution. The model performance may not reflect the best performance presented in paper.

What dataset has the model been trained on in this pretrained file?
